action.yml

```
@@ -377,9 +387,9 @@ runs:
             exit 1
           fi
 
-          # Continue if there's nothing to commit
-          if [[ $GIT_STATUS == "" ]]; then
-            echo "No changes found when merging-up $FROM_BRANCH into $INTO_BRANCH. Skipping."
+          # Things are update to up to date, so do not commit
+          if [[ $MERGE_RESULT =~ "Already up to date" ]]; then
+            echo "Merge result said things are already up to date. Skipping."
```

```suggestion
            echo "Merge result said things are already up to date when merging-up $FROM_BRANCH into $INTO_BRANCH. Skipping."
```
I've found myself using that part of the log quite a bit, so I'd like to retain it.

---

Updated